src/main/resources/assets/neoforge/lang/en_us.json

```diff
@@ -119,6 +119,14 @@
   "commands.neoforge.timespeed.query.default": "Time in %s flows normally (20 minutes per day).",
   "commands.neoforge.timespeed.set": "Set flow of time in %s to %sx (%s minutes per day).",
   "commands.neoforge.timespeed.set.default": "Set flow of time in %s to default (20 minutes per day).",
+  "commands.neoforge.data_components.list.error.held_stack_empty": "Held stack must not be empty",
```

Comment:
This translation could probably be changed to something like "You are not holding any item", to be consistent with `commands.enchant.failed.itemless`.